source/keyboard-shortcuts.txt

```diff
@@ -0,0 +1,247 @@
+==================
+Keyboard Shortcuts
+==================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Keyboard shortcuts enable you to easily navigate |compass-short|.
+
+.. list-table::
+   :widths: 20 30 25 25
+   :header-rows: 1
+
+   * - Category
+     - Description
+     - Windows
+     - Mac
+
+   * - Menu
+     - Online help
+     - ``F1``
+     - ``F1``
+
+   * - Menu
+     - Quit application
+     - ``Ctrl`` + ``Q``
+     - ``Cmd`` + ``Q``
+
+   * - Menu
+     - Show settings
+     - ``Ctrl`` + ``,``
+     - ``Cmd`` + ``,``
+
+   * - Menu
+     - Hide window
+     - ``Ctrl`` + ``H``
+     - ``Cmd`` + ``H``
+
+   * - Menu
+     - Hide other windows
+     - ``Ctrl`` + ``Shift`` + ``H``
+     - ``Cmd`` + ``Shift`` + ``H``
+
+   * - Menu
+     - Opens a new Compass window
+     - ``Ctrl`` + ``N``
+     - ``Cmd`` + ``N``
+
+   * - Menu
+     - Share schema as JSON
+     - ``Alt`` + ``Ctrl`` + ``S``
+     - ``Alt`` + ``Cmd`` + ``S``
+
+   * - Menu
+     - Reload screen
+     - ``Ctrl`` + ``Shift`` + ``R``
+     - ``Cmd`` + ``Shift`` + ``R``
+
+   * - Menu
+     - Reload data
+     - ``Ctrl`` + ``R``
+     - ``Cmd`` + ``R``
+
+   * - Menu
+     - Toggle sidebar
+     - ``Ctrl`` + ``Shift`` + ``D``
+     - ``Cmd`` + ``Shift`` + ``D``
+
+   * - Menu
+     - Actual size
+     - ``Ctrl`` + ``0``
+     - ``Cmd`` + ``0``
+
+   * - Menu
+     - Zoom in
+     - ``Ctrl`` + ``=``
+     - ``Cmd`` + ``=``
+
+   * - Menu
+     - Zoom out
+     - ``Ctrl`` + ``-``
+     - ``Cmd`` + ``-``
+
+   * - Menu
+     - Toggle DevTools
+     - ``Alt`` + ``Ctrl`` + ``I``
+     - ``Alt`` + ``Cmd`` + ``I``
+
+   * - Menu
+     - Minimize
+     - ``Ctrl`` + ``M``
+     - ``Cmd`` + ``M``
+
+   * - General
+     - Undo
+     - ``Ctrl`` + ``Z``
+     - ``Cmd`` + ``Z``
+
+   * - General
+     - Redo
+     - ``Ctrl`` + ``Z``
+     - ``Cmd`` + ``Z``
+
+   * - General
+     - Cut
+     - ``Ctrl`` + ``X``
+     - ``Cmd`` + ``X``
+
+   * - General
+     - Copy
+     - ``Ctrl`` + ``C``
+     - ``Cmd`` + ``C``
+
+   * - General
+     - Paste
+     - ``Ctrl + V``
+     - ``Cmd + V``
+
+   * - General
+     - Select all
+     - ``Ctrl`` + ``A``
+     - ``Cmd`` + ``A``
+
+   * - General
+     - Find
+     - ``Ctrl`` + ``F``
+     - ``Cmd`` + ``F``
+
+   * - General
+     - Navigate to next tab
+     - ``Tab``
+     - ``Tab``
+
+   * - General
+     - Navigate to previous tab
+     - ``Shift`` + ``Tab``
+     - ``Shift`` + ``Tab``
+
```

Comment:
I don't think we have a hotkey like this.
```suggestion
```

Reply:
Removed.